packages/modules/src/modules/banners/choiceCardsBanner/ChoiceCardsBanner.stories.tsx

```diff
@@ -15,7 +15,7 @@ import { PageTracking, TestTracking, Tracking } from '@sdc/shared/src/types';
 
 export default {
     component: ChoiceCardsBanner,
-    title: 'Banners/Subscriptions/ChoiceCardsBanner',
```

Comment:
What’s the reason for this change, exactly?

Reply:
It took me a while to find the banner when I was checking in storybook - not sure it belongs under 'subscriptions'?

Reply:
Ah, I see – wasn’t sure what the naming affected. Happy with your judgement!